python311Packages.ical: 5.1.1 -> 6.1.0

fabaff opened the pull request:
Diff: https://github.com/allenporter/ical/compare/refs/tags/5.1.1...6.0.0

Changelog: https://github.com/allenporter/ical/releases/tag/6.0.0

## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff committed:
python311Packages.ical: 5.1.1 -> 6.0.0

Diff: https://github.com/allenporter/ical/compare/refs/tags/5.1.1...6.0.0

Changelog: https://github.com/allenporter/ical/releases/tag/6.0.0

fabaff commented:
Result of `nixpkgs-review pr 265796` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages failed to build:</summary>
  <ul>
    <li>home-assistant-component-tests.google</li>
    <li>python310Packages.gcal-sync</li>
    <li>python310Packages.gcal-sync.dist</li>
    <li>python311Packages.gcal-sync</li>
    <li>python311Packages.gcal-sync.dist</li>
  </ul>
</details>
<details>
  <summary>11 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.local_calendar</li>
    <li>home-assistant-component-tests.local_todo</li>
    <li>home-assistant-component-tests.rainbird</li>
    <li>python310Packages.ical</li>
    <li>python310Packages.ical.dist</li>
    <li>python310Packages.pyrainbird</li>
    <li>python310Packages.pyrainbird.dist</li>
    <li>python311Packages.ical</li>
    <li>python311Packages.ical.dist</li>
    <li>python311Packages.pyrainbird</li>
    <li>python311Packages.pyrainbird.dist</li>
  </ul>
</details>

dotlambda reviewed (changes requested):
Remove dependency emoji.

fabaff committed:
python311Packages.ical: 6.0.0 -> 6.1.0

Diff: allenporter/ical@refs/tags/6.0.0...6.1.0

Changelog: https://github.com/allenporter/ical/releases/tag/6.1.0

fabaff committed:
python311Packages.gcal-sync: 5.0.0 -> 6.0.1

Changelog: https://github.com/allenporter/gcal_sync/releases/tag/6.0.1

fabaff commented:
Result of `nixpkgs-review pr 265796` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>16 packages failed to build:</summary>
  <ul>
    <li>home-assistant-component-tests.google</li>
    <li>home-assistant-component-tests.local_calendar</li>
    <li>home-assistant-component-tests.local_todo</li>
    <li>home-assistant-component-tests.rainbird</li>
    <li>python310Packages.gcal-sync</li>
    <li>python310Packages.gcal-sync.dist</li>
    <li>python310Packages.ical</li>
    <li>python310Packages.ical.dist</li>
    <li>python310Packages.pyrainbird</li>
    <li>python310Packages.pyrainbird.dist</li>
    <li>python311Packages.gcal-sync</li>
    <li>python311Packages.gcal-sync.dist</li>
    <li>python311Packages.ical</li>
    <li>python311Packages.ical.dist</li>
    <li>python311Packages.pyrainbird</li>
    <li>python311Packages.pyrainbird.dist</li>
  </ul>
</details>

fabaff commented:
Was done with #269229
